lib/maily/email.rb

```diff
@@ -105,5 +109,24 @@ def update_template(new_content, part = nil)
         f.write(new_content)
       end
     end
+
+    def versions
+      regexp = Regexp.new("^#{self.name}:")
+      self.mailer.emails.select do |email_key, _email|
+        email_key.match(regexp)
+      end
+    end
+
+    class << self
+      def name_with_version(name, version = nil)
+        _version = formatted_version(version)
+        [name, _version].join(':')
+      end
+
+      def formatted_version(version)
+        _version = version.presence || DEFAULT_VERSION
+        _version.try(:parameterize).try(:underscore)
```

Comment:
instead of `try`, why not `&.`?

Reply:
because of this comment :)

https://github.com/markets/maily/pull/31#issuecomment-426754998

_"`(3)` We still support Ruby 2.2, so we should try to avoid using `&.`"_

Reply:
ah ok! that's a very very old comment 😄  we support now Ruby 2.5+ 
let's do it!